### PR TITLE
redesign campaign index page

### PR DESCRIPTION
**Pull request by lopezabegaillyn**
- Adds Campaign sorting to campaigns.js
- Creates table for the campaigns in the Campaign index page
- Sorts the campaigns in alphabetical order

### CODE REVIEW

File: config/application.example.yml [hunk 1]
```diff
@@ -1,139 +0,0 @@
...
-# Username of user who will receive requests for technical help from the Get Help button
-# Can be left blank.
-  technical_help_staff: 'Sage (Wiki Ed)'
-
-# Rocket.Chat integration for courses, which requires access to an admin
-# account on the Rocket.Chat server.
-  enable_chat: 'false'
-  chat_server: 'https://your.rocket.chat'
-  chat_admin_username: 'admin'
-  chat_admin_password: 'passwiord'
```
Comment (medhabansal) [inline config/application.example.yml]: @lopezabegaillyn .example files need to be retained in the project repository.

File: config/database.example.yml [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
-  database: dashboard_testing
-
-staging:
-  <<: *default
-  database: <staging-database-name>
-
-production:
-  <<: *default
-  database: <production-database-name>
-  pool: 40
```
Comment (medhabansal) [inline config/database.example.yml]: We need this .example.yml file as well.

**Comment by lopezabegaillyn** [comment]
Hi @medhabansal 
I've already added back the *.example.yml files 
Thank you!

**Comment by ragesoss** [comment]
@medhabansal let me know when you've had a chance to try it out locally.

**Comment by medhabansal** [comment]
@ragesoss Tested locally, works pretty well. Attaching screenshot of redesigned page.
after_redesign

**Comment by ragesoss** [comment]
Thanks! This looks like a good start on #1126; having the table structure in place will make it easy to gradually improve the functionality of that page. Eventually, we probably want something that combines the features of this page (the CSVs) with the table on the explore page (which has a lot more info and sortable columns, and can act as a comparative summary of the various campaigns).

**Comment by lopezabegaillyn** [comment]
Thank you @medhabansal and @ragesoss

**Pull request merged**